en/middleware/micrortps.md

```diff
@@ -45,18 +50,22 @@ The main elements of the architecture are the client and agent processes shown i
   In a typical configuration they will both be on the same system (e.g. a development computer, Linux companion computer or compute board), connected to the *Client* over a Wifi link or via USB.
 
 
-## Architectural overview for a ROS2/ROS application pipeline
+### ROS2/ROS application pipeline
+
```

Comment:
FYI, this had major change to make it clear what the pipeline looks like in both cases.